[skills] Update Scout namespace paths after security_solution test reorganisation (#275087) (#280060)
elastic/kibana: x-pack/solutions/security/plugins/security_solution/.agents/skills/scout-best-practices-reviewer/SKILL.md

### Test placement

Security Solution Scout tests use a namespace sub-directory structure. Each feature area has its own directory under `test/scout/`:

| Namespace | Path |
|-----------|------|
| `agent_builder` | `test/scout/agent_builder/ui/` |
| `entity_analytics` | `test/scout/entity_analytics/{ui,api}/` |
| `exceptions` | `test/scout/exceptions/ui/` |
| `flyout` | `test/scout/flyout/ui/` |
| `reports` | `test/scout/reports/ui/` |
| `timelines` | `test/scout/timelines/ui/` |
| `workflows` | `test/scout/workflows/ui/` |

- Timeline UI tests belong in `security_solution/test/scout/timelines/ui/` — the `timelines` plugin only has server-side saved object definitions and APIs
- Parallel specs go in `test/scout/<namespace>/ui/parallel_tests/`
- Sequential specs go in `test/scout/<namespace>/ui/tests/`

> - Root-level `test/scout/ui/` contains only `.scout/reports` output, not test specs; `test/scout/api/` does not exist — all test specs live under namespace sub-dirs

Reviewer: **Factual bug.** `test/scout/ui/` does not exist in the repo — the claim about `.scout/reports` output is not accurate:

```bash
$ ls x-pack/solutions/security/plugins/security_solution/test/scout/ui/
ls: No such file or directory

$ find x-pack/solutions/security/plugins/security_solution/test/scout -name ".scout" -type d
(no results)
```

`.scout/reports` is Playwright runtime output — not a committed directory. The pre-PR wording (*"root-level `ui/` no longer exists"*) was actually accurate. Suggested rewording:

```suggestion
- There is no root-level `test/scout/ui/` or `test/scout/api/` in `security_solution` — all test specs live under namespace sub-dirs
```
